x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/components/back_to_endpoint_details_flyout_subheader.tsx

```diff
@@ -28,24 +36,32 @@ export const BackToEndpointDetailsFlyoutSubHeader = memo<{ endpointId: string }>
         }),
       [currentUrlQueryParams, endpointId]
     );
-
     const backToDetailsClickHandler = useNavigateByRouterEventHandler(detailsRoutePath);
-
-    const backButtonProp = useMemo((): FlyoutSubHeaderProps['backButton'] => {
+    const backButtonProps = useMemo((): BackButtonProps['backButton'] => {
       return {
         title: i18n.translate('xpack.securitySolution.endpoint.policyResponse.backLinkTitle', {
-          defaultMessage: 'Endpoint Details',
+          defaultMessage: 'Endpoint details',
         }),
         href: getAppUrl({ path: detailsRoutePath }),
         onClick: backToDetailsClickHandler,
       };
     }, [backToDetailsClickHandler, getAppUrl, detailsRoutePath]);
 
     return (
-      <FlyoutSubHeader
-        backButton={backButtonProp}
-        data-test-subj="endpointDetailsPolicyResponseFlyoutHeader"
-      />
+      <div className="buttons">
```

Comment:
Same here, why do you need this class name?

Reply:
53ad761a43d1aa0101049777c49270b398e8b08e